monai/data/utils.py

```diff
@@ -547,6 +548,30 @@ def set_rnd(obj, seed: int) -> int:
     return seed
 
 
+def affine_to_spacing(affine: NdarrayTensor, dims: int = 3, dtype=float, suppress_zeros: bool = True) -> NdarrayTensor:
+    """
+    Computing the current spacing from the affine matrix.
+
+    Args:
+        affine: a d x d affine matrix.
+        dims: indexing of the spatial dimensions `affine[:dims, :dims]`.
```

Comment:
Maybe we should use arg name `ndim` to be consistent with others?

Thanks.

Reply:
it's not actually ndim, it could be `-1` here for the indexing.

Reply:
If so, I would suggest to change another arg name, `dims` looks like a list of something, confusing.

Thanks.

Reply:
sure updated to `r`.